cypress/common/selectors.ts

```diff
@@ -101,6 +101,7 @@ export const alertMessages = {
 }
 
 export const block = {
-    differentReactVersionsIsolatedSharedBlock: '[data-e2e="SHARED__REACT_BLOCK"]'
+    differentReactVersionsIsolatedSharedBlock: '[data-e2e="SHARED__REACT_BLOCK"]',
+    reactDivBlock: '[data-e2e="REACT__DIV_BLOCK"]',
```

Comment:
can be merged with button block and get div/button as replace param

Reply:
Fixed